packages/block-editor/src/components/block-preview/index.js

```diff
@@ -37,7 +37,6 @@ export function BlockPreview( {
 	);
 	const settings = useMemo( () => {
 		const _settings = { ...originalSettings };
-		_settings.__experimentalBlockPatterns = [];
```

Comment:
Thanks for contributing, @inc2734.

It looks like this was intentionally added to avoid parsing patterns in block previews - https://github.com/WordPress/gutenberg/pull/29444#discussion_r591526310.

Unfortunately, I'm not familiar with this part of the code, so maybe folks who reviewed the original PR can help us out. Cc @mcsf, @ntsekouras

Reply:
Thanks for the PR @inc2734! @Mamaduka is right about the reasoning for this line of code. We should try to find some different solution here.